blocklylib-core/src/main/java/com/google/blockly/android/WorkspaceUiInterface.java

```diff
@@ -0,0 +1,17 @@
+package com.google.blockly.android;
+
+import com.google.blockly.android.control.BlocklyController;
+
+/**
+ * An interface that specifies the actions that can be taken on a workspace ui component.
+ */
+public interface WorkspaceUiInterface {
```

Comment:
This one probably isn't useful, unless we are replace all of the methods to manipulate BlockViews. We aren't encouraging that, since it would eliminate the vast majority of what Blockly infrastructure provides.